src/design-system/context/DesignSystemContext.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { ColorMode, ColorModeProvider } from '../color/ColorMode';
+import {
+  ExperimentalFlags,
+  ExperimentalFlagsProvider,
+} from './ExperimentalFlagsContext';
+
+type DesignSystemContextValue = {
+  experimentalFlags: {
+    androidShadowsV2?: boolean;
+  };
+};
+
+const DesignSystemContext = React.createContext<DesignSystemContextValue>({
+  experimentalFlags: {
+    androidShadowsV2: false,
+  },
+});
+
+export function DesignSystemProvider({
+  children,
+  colorMode,
+  experimentalFlags = {},
```

Comment:
This default value should be hoisted out of the function or it'll be a new context value on every render.

Reply:
Or we could make this a non-issue by memoizing the context value within `ExperimentalFlagsProvider` itself.